### DISCUSSION TITLE
Partial address matching

### DISCUSSION THREAD

**Discussion by SxTx86** [osm-search/Nominatim#2680] [General]
Hello,

We are using Nomination Api in our project.
The response we receive looks like this:
https://nominatim.openstreetmap.org/search?q=23-25,%20via%20ardea&format=geocodejson&accept-language=en&polygon=1&addressdetails=1
Is it possible to find out if the address is "partial_match" as in Google?

Many thanks

**Comment by lonvia** [comment]
Usually the "type" field should tell you what level of result (house, street, etc.), see [geocodejson spec](https://github.com/geocoders/geocodejson-spec/tree/master/draft). You should be able to infer from that if the whole address was found or only a part. I see that Nominatim is currently doing it wrong. Needs fixing.
